arch/xtensa/src/esp32/esp32_openeth.c

```diff
@@ -772,21 +468,14 @@ int esp32_openeth_initialize(void)
       abort();
     }
 
-  /* Get the interface structure associated with this interface number. */
+  /* Initialize the driver structure */
 
-  priv = &g_openeth;
+  memset(priv, 0, sizeof(struct openeth_priv_s));
 
-  /* Initialize the driver structure */
+  dev->ops = &g_ops;
 
-  memset(priv, 0, sizeof(struct openeth_driver_s));
-  priv->dev.d_ifup = openeth_ifup;       /* I/F up (new IP address) callback */
-  priv->dev.d_ifdown = openeth_ifdown;   /* I/F down callback */
-  priv->dev.d_txavail = openeth_txavail; /* New TX data callback */
-#ifdef CONFIG_NET_MCASTGROUP
-  priv->dev.d_addmac = openeth_addmac; /* Add multicast MAC address */
-  priv->dev.d_rmmac = openeth_rmmac;   /* Remove multicast MAC address */
-#endif
-  priv->dev.d_private = priv; /* Used to recover private state from dev */
+  dev->quota[NETPKT_TX] = 1;
+  dev->quota[NETPKT_RX] = 1;
```

Comment:
Why to hard-code quota to 1? Please include some comments here

Reply:
I will update this so it is not hard coded and will add proper handling of multiple TX buffers.

Reply:
done